composer.json

```diff
@@ -5,42 +5,26 @@
   "description": "Mautic Open Source Distribution",
   "require": {
     "composer/installers": "^1.10",
-    "mautic/core-lib": "self.version"
-  },
-  "replace": {
-    "mautic/plugin-citrix": "self.version",
-    "mautic/plugin-clearbit": "self.version",
-    "mautic/plugin-cloudstorage": "self.version",
-    "mautic/plugin-crm": "self.version",
-    "mautic/plugin-emailmarketing": "self.version",
-    "mautic/plugin-focus": "self.version",
-    "mautic/plugin-fullcontact": "self.version",
-    "mautic/plugin-gmail": "self.version",
-    "mautic/plugin-outlook": "self.version",
-    "mautic/plugin-social": "self.version",
-    "mautic/plugin-zapier": "self.version",
-    "mautic/theme-coffee": "self.version",
-    "mautic/theme-mauve": "self.version",
-    "mautic/theme-nature": "self.version",
-    "mautic/theme-aurora": "self.version",
-    "mautic/theme-blank": "self.version",
-    "mautic/theme-blank-grapejs": "self.version",
-    "mautic/theme-brienz": "self.version",
-    "mautic/theme-cards": "self.version",
-    "mautic/theme-confirmme": "self.version",
-    "mautic/theme-fresh-center": "self.version",
-    "mautic/theme-fresh-fixed": "self.version",
-    "mautic/theme-fresh-left": "self.version",
-    "mautic/theme-fresh-wide": "self.version",
-    "mautic/theme-goldstar": "self.version",
-    "mautic/theme-neopolitan": "self.version",
-    "mautic/theme-oxygen": "self.version",
-    "mautic/theme-paprika": "self.version",
-    "mautic/theme-skyline": "self.version",
-    "mautic/theme-sparse": "self.version",
-    "mautic/theme-sunday": "self.version",
-    "mautic/theme-vibrant": "self.version"
+    "mautic/core-lib": "self.version",
+    "mautic/grapes-js-builder-bundle": "dev-do-not-require-composer-plugin"
   },
+  "require-dev": {
+    "symfony/web-profiler-bundle": "~4.4.0",
+    "symfony/var-dumper": "~4.4.0",
+    "symfony/browser-kit": "~4.4.0",
+    "symfony/dom-crawler": "~4.4.0",
+    "mautic/transifex": "^4.0",
+    "http-interop/http-factory-guzzle": "^1.0",
+    "webfactory/exceptions-bundle": "~4.3",
+    "friendsofphp/php-cs-fixer": "~2.16.1",
+    "liip/test-fixtures-bundle": "^1.6",
+    "phpstan/phpstan": "^0.12.82",
```

Comment:
Just quickly checking on my phone but could we leave PHPSTAN at its old version 0.12.25 as it is now? With higher versions we had a plethora of issues in combination with Rector, so that causes the failing checks here I believe. We can't merge it until all CI is green sorry ;)

Reply:
Afaik I didn't change any version, but I'll change it. Curious where it came from